### ISSUE TITLE
[FEATURE] Horizontal option for Navigation Buttons

### ISSUE THREAD

**Issue by christiansass** [bitfocus/companion#1673]
![Settings](https://user-images.githubusercontent.com/35471675/126805980-fe1e2db4-8948-4bee-858a-7a0398c8488f.png)

I think it would be beneficial to allow users to set the Navigation Buttons to render across the top or bottom of the stream deck Horizontally instead of vertically and possibly set the corner or key to start from. Currently if you want to do this you have to change ALL 99 pages. I'm assuming this could be implemented the same way you change or reverse the navigation buttons.

currently the existing implementation only leave the bottom row fully available.
![Example](https://user-images.githubusercontent.com/35471675/126807012-f83185fa-a9ae-44e6-94de-5036ea1c8478.png)

The main reason for requesting this is i would like to use the two bottom rows of the stream deck XL for mutes on a 16 channel audio mixer,. Im sure others could use this for other customizations as well. 
If this were made available it would also allow two rows along the bottom of a standard streamdeck or a nice horizontal layout for the mini

alternatively i was thinking about some sort of two key navigation since the middle key is only being used for an indicator. Which could also greatly benefit the mini users

**Comment by Julusian** [comment]
I dont think this will work as a config option there. The existing options change how the buttons look and not where they are positioned in the grid.
This will need to be a button somewhere, as it is an operation to be performed once to move them all (or move them on one page?)

**Comment by Julusian** [comment]
Merging this as a duplicate of #880
